modules/trusted-firmware-m/Kconfig

```diff
@@ -466,4 +466,11 @@ config TFM_NRF_PROVISIONING
 	  provisioning lifecycle state in order to boot. See provisioning
 	  guide for more details.
 
+config TFM_PSA_FRAMEWORK_HAS_MM_IOVEC
+	bool
+	prompt "TF-M MM-IOVEC" if TFM_SFN
```

Comment:
I couldn't find a lot of documentation for this configuration, but from what I can tell it is not limited to SFN, but to isolation level 1.

There is a configuration check that when this is enabled the isolation level is not larger than 1.

Also, depends on seems better than prompt if checking.

Reply:
I can update to isolation level 1, sure.